### PR TITLE
Delegated-admin-extension docs

### PR DESCRIPTION
**Pull request by zxan1285**
- new rule sample to use namespaced claims
 - added "Auditor" role description. this role was added in v3

### CODE REVIEW

**Comment by jeffreylees** [comment]
This PR is waiting for https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/116, correct?

**Comment by zxan1285** [comment]
@jeffreylees yes, new rule will work only with changes from https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/116

**Comment by khorne3** [comment]
@zxan1285 I'll flag this as `do-not-merge`, but please feel free to let us know when we should merge this in.

File: articles/extensions/delegated-admin/v2/index.md [hunk 1]
```diff
@@ -118,13 +117,12 @@ This rule gives users from the `IT Department` the `Delegated Admin - Administra
 ```js
 function (user, context, callback) {
  if (context.clientID === 'CLIENT_ID') {
+   const namespace = 'https://tenant-name.us8.webtask.io/auth0-delegated-admin';
```
Comment (fyockm) [inline articles/extensions/delegated-admin/v2/index.md]: @jeffreylees should we use `${account.tenant}` here instead? does that work within a code snippet?

File: articles/extensions/delegated-admin/v3/index.md [hunk 2]
```diff
@@ -134,13 +135,12 @@ This rule gives users from the `IT Department` the `Delegated Admin - Administra
 ```js
 function (user, context, callback) {
  if (context.clientID === 'CLIENT_ID') {
+   const namespace = 'https://tenant-name.us8.webtask.io/auth0-delegated-admin';
```
Comment (fyockm) [inline articles/extensions/delegated-admin/v3/index.md]: as above, should we also use `${account.tenant}` here instead?

**Review by fyockm — COMMENTED** [review]
@zxan1285 Can we also update the urls under "Configure Application Settings" to point to Node 8?

File: articles/extensions/delegated-admin/v2/index.md (hunk 1 above)
Comment (jeffreylees) [reply articles/extensions/delegated-admin/v2/index.md]: Yes

**Review by fyockm — APPROVED** [review]

**Comment by shawnmclean** [comment]
Hi @khorne3, we've released this change in DAE v3.2.0. This should be able to be merged in now: https://github.com/auth0-extensions/auth0-delegated-administration-extension/releases/tag/v3.2.0

**Pull request merged**